redisinsight/ui/src/components/main-router/config.ts

```diff
@@ -0,0 +1,14 @@
+import { isArray } from 'lodash'
+import { IRoute } from 'uiSrc/constants'
+
+export const LAZY_LOAD = window.riConfig?.app?.lazyLoad
```

Comment:
@ArtemHoruzhenko Is this the right way to use the new config?

Reply:
yes